### PR TITLE
Fix verifing signed payload

### PR DESCRIPTION
**Pull request by githinho**

### CODE REVIEW

File: src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt [hunk 1]
```diff
@@ -51,6 +53,10 @@ class VerificationServiceImpl : VerificationService {
         }
     }
 
+    private fun generateEip191Message(message: ByteArray): ByteArray =
+        0x19.toByte().toByteArray() + 0x45.toByte().toByteArray() +
+            ("thereum Signed Message:\n" + message.size).toByteArray() + message
```
Comment (domagojlatecki) [inline src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt:58]: "Ethereum" not "thereum"

**Review by domagojlatecki — APPROVED** [review]

File: src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt (hunk 1 above)
Comment (githinho) [reply src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt:58]: This is copied from the library:
https://github.com/komputing/KEthereum/blob/master/eip191/src/main/kotlin/org/kethereum/eip191/EIP191.kt

File: src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt [hunk 2]
```diff
@@ -51,6 +53,10 @@ class VerificationServiceImpl : VerificationService {
         }
     }
 
+    private fun generateEip191Message(message: ByteArray): ByteArray =
+        0x19.toByte().toByteArray() + 0x45.toByte().toByteArray() +
```
[the remark this answers is not in the document]
Comment (githinho) [reply src/main/kotlin/com/ampnet/identityservice/service/impl/VerificationServiceImpl.kt:57]: This is copied from the library:
https://github.com/komputing/KEthereum/blob/master/eip191/src/main/kotlin/org/kethereum/eip191/EIP191.kt

**Review by josipkoncurat — APPROVED** [review]
Hope it works fine with the frontend!

**Pull request merged**